app/scenes/mfa/mfa.js

```diff
@@ -67,40 +76,48 @@ export default class Mfa extends Component {
         return (
             <KeyboardAvoidingView
                 behavior='padding'
-                style={[GlobalStyles.container, GlobalStyles.signupContainer]}
+                style={{flex: 1}}
+                keyboardVerticalOffset={0}
             >
-                <Image
-                    source={logo}
-                />
-                <FormattedText
-                    style={[GlobalStyles.header, GlobalStyles.label]}
-                    id='login_mfa.enterToken'
-                    defaultMessage="To complete the sign in process, please enter a token from your smartphone's authenticator"
-                />
-                <ErrorText error={this.state.error}/>
-                <TextInputWithLocalizedPlaceholder
-                    value={this.state.token}
-                    onChangeText={this.handleInput}
-                    onSubmitEditing={this.submit}
-                    style={GlobalStyles.inputBox}
-                    autoCapitalize='none'
-                    autoCorrect={false}
-                    keyboardType='numeric'
-                    placeholder={{id: 'login_mfa.token', defaultMessage: 'MFA Token'}}
-                    returnKeyType='go'
-                    underlineColorAndroid='transparent'
-                />
-                <Button
-                    onPress={this.submit}
-                    loading={false}
-                    containerStyle={GlobalStyles.signupButton}
-                >
-                    <FormattedText
-                        style={GlobalStyles.signupButtonText}
-                        id='mobile.components.select_server_view.proceed'
-                        defaultMessage='Proceed'
-                    />
-                </Button>
+                <TouchableWithoutFeedback onPress={this.blur}>
```

Comment:
Two things scrollViews aren't very performant and the second thing is that I wanted to be able to tap the screen and execute the blur function so that way the keyboard gets hidden and there's no need to give other type of feedback to the user.